source/Symbol/SwiftASTContext.cpp

```diff
@@ -1468,18 +1468,30 @@ void SwiftASTContext::RemapClangImporterOptions(
                   remapped.c_str());
     options.BridgingHeader = remapped;
   }
+  bool remap_next = false;
   for (auto &arg_string : options.ExtraArgs) {
     StringRef prefix;
     StringRef arg = arg_string;
-    if (arg.consume_front("-I"))
+
+    if (!remap_next) {
+      /* previous argument was -iquote */;
+    } else if (arg == "-I" || arg == "-F" || arg == "-iquote") {
```

Comment:
Clang allows `-iquote` to be used in joined form as well (e.g., `-iquotefoo/bar` is legal): https://github.com/llvm/llvm-project/blob/fcbec02/clang/include/clang/Driver/Options.td#L1951-L1952

Does that case need to be handled here as well? It's ugly, but we actually use it in our Swift Bazel rules to remove some excess `<space>-Xcc<space>`s in our command lines.

Reply:
Done.